### PR TITLE
Auto make coverage 

### LINKED ISSUE #248: CI/CD Managements in VATZ(VATZ, VATZ-PROTO)
## Checklist
- [ ] New Feature for the SVC/Plugin
- [x] New Feature for CI/CD
- [ ] Enhancement on Vatz
- [ ] Others(etc. e.g, documentation,...)
---

> This is an issue to manage CI/CD managements in VATZ. 
any discussion or need help need on this, will be raised or discussed on this issue thread.

### PR DESCRIPTION
**Pull request by xellos00**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [x] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [ ] others (anything other than above)

---

**Related:** # (issue)
close #248
**Summary**

---

### CODE REVIEW

**Review by rootwarp — APPROVED** [review]
LGTM

**Review by heejin-github — APPROVED** [review]
LGTM

File: .github/workflows/ci_coverage.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  coverage:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
```
Comment (meetrick) [inline .github/workflows/ci_coverage.yml]: You changed the python version from `3.10` to `3.x`, right?

**Review by meetrick — APPROVED** [review]
LGTM

**Pull request merged**